vae_medmnist/evaluation/evaluation.py

```diff
@@ -186,6 +206,10 @@ def save_reconstructions(model, dataloader, device, save_path, descriptions):
 
     datamodule.setup()
 
+    # Try to get labels from an instance of the dataset class
+    dataset_instance = datasetclass(split='train', download=True)
```

Comment:
remove, if you want a single dataset you can call the accumulated dataset with a list of ['datasetname'] and query the labers via the class